auth-web/src/components/pay/eft/ShortNameAccountLink.vue

```diff
@@ -203,6 +235,17 @@ export default defineComponent({
       return `${props.shortNameDetails.accountId} ${props.shortNameDetails.accountName}`
     })
 
+    async function evaluateLinks () {
+      let pending = []
+      state.results.map(result => {
+        if (result.statusCode === 'PENDING' && result.amountOwing > 0) {
```

Comment:
ENUM

Reply:
```
let pending = state.results.filter(result => 
  result.statusCode === 'PENDING' && result.amountOwing > 0
);

await Vue.nextTick();
state.expanded = [...pending];
```

Reply:
ENUM and code update, thanks much cleaner